src/pe/header.rs

```diff
@@ -569,6 +570,30 @@ impl Header {
             optional_header,
         })
     }
+
+    /// Parses PE header from the given bytes that not contains DOS stub, default DosHeader will be used
```

Comment:
grammar: Parses PE header from the given bytes, a default DosHeader and DosStub are generated, and any malformed header or stub is ignored.

Reply:
thanks for suggestion, fixed